### ISSUE TITLE
[BUG] Selenium error

### ISSUE THREAD

**Issue by Flavien06** [charlesbel/Microsoft-Rewards-Farmer#91]
Hello, I have this error:
**Error Log**
```
Traceback (most recent call last):
  File "ms_rewards_farmer.py", line 771, in <module>
    browser = browserSetup(True, PC_USER_AGENT)
  File "ms_rewards_farmer.py", line 35, in browserSetup
    chrome_browser_obj = webdriver.Chrome(options=options)
  File "/home/flavien/.local/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py", line 73, in __init__
    self.service.start()
  File "/home/flavien/.local/lib/python3.8/site-packages/selenium/webdriver/common/service.py", line 98, in start
    self.assert_process_still_running()
  File "/home/flavien/.local/lib/python3.8/site-packages/selenium/webdriver/common/service.py", line 109, in assert_process_still_running
    raise WebDriverException(
selenium.common.exceptions.WebDriverException: Message: Service chromedriver unexpectedly exited. Status code was: 1
```

**Comment by Dangman368** [comment]
here you go: go to requirements.txt replace selenium with selenium==3.14.1 then do "pip uninstall -r requirements.txt" after everything uninstalls then do "pip install -r requirements.txt"

**Comment by FRKFFR** [comment]
someone re created the bot https://github.com/MehdiRtal/MicrosoftRewardsBot.git

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
